src/OrbitGl/EventTrack.cpp

```diff
@@ -251,7 +242,7 @@ std::string EventTrack::GetSampleTooltip(PickingId id) const {
   static const std::string unknown_return_text = "Function call information missing";
 
   auto user_data = time_graph_->GetBatcher().GetUserData(id);
-  if (!user_data || !user_data->custom_data_) {
+  if (user_data == nullptr || user_data->custom_data_ == nullptr) {
```

Comment:
Thanks, but I guess we should "briefly" rediscuss. AFAIK we have not decided on anything here, or well we said both is fine. But I know that clang-tidy complains.

Reply:
Agree, but then let us decide on this! We should bring this up in our next meeting.